--- Imaging-1.1.4/Setup.in.wiget	2003-04-22 18:11:25.000000000 +0200
+++ Imaging-1.1.4/Setup.in	2003-07-23 11:54:34.000000000 +0200
@@ -42,7 +42,7 @@
 
 _imagingtk  _imagingtk.c \
 	-IlibImaging Tk/tkImaging.c \
-	-I/usr/local/include -L/usr/local/lib -ltcl8.3 -ltk8.3 \
+	-I/usr/local/include -L/usr/local/lib -ltcl -ltk \
         -L/usr/X11R6/lib -lX11 
 
 # --------------------------------------------------------------------
