--- Imaging-1.1.7/PIL/ImageShow.py~	2009-11-01 01:44:12.000000000 +0100
+++ Imaging-1.1.7/PIL/ImageShow.py	2010-02-16 00:00:20.623473929 +0100
@@ -145,6 +145,15 @@
     if which("display"):
         register(DisplayViewer)
 
+    class KFMClientViewer(UnixViewer):
+        def get_command_ex(self, file, **options):
+            executable = "kfmclient"
+            command = executable + " exec"
+            return command, executable
+
+    if which("kfmclient"):
+        register(KFMClientViewer)
+
     class XVViewer(UnixViewer):
         def get_command_ex(self, file, title=None, **options):
             # note: xv is pretty outdated.  most modern systems have
